src/http/api/feed.ts

```diff
@@ -55,24 +47,62 @@ export function createGetFeedHandler(
             });
         }
 
-        // Get feed items
-        const { items: feedItems, nextCursor } =
-            await feedService.getFeedFromKvStore(db, apCtx, {
-                postType,
-                limit,
-                cursor,
-            });
-
-        // Prepare response
-        const posts: Post[] = [];
+        const account = await accountService.getDefaultAccountForSite(
+            ctx.get('site'),
+        );
 
-        for (const item of feedItems) {
-            const post = await mapActivityToPost(item, accountService, apCtx);
+        const { results, nextCursor } = await feedService.getFeedData({
+            accountId: account.id,
+            postType,
+            limit,
+            cursor,
+        });
 
-            if (post) {
-                posts.push(post);
-            }
-        }
+        const posts: Post[] = results.map((result) => {
+            return {
```

Comment:
I think I would like to see this mapping happen in the FeedService 🤔

Reply:
This logic is dealing with mapping data to a response returned by the API, it feels to me that this mapping should reside at this layer. I suppose its unlikely that `FeedService.getFeedData` is going to be used anywhere else though 🤔